flow-defs/generated/mistica-icons/icon-boltons-filled.js.flow

```diff
@@ -5,5 +5,5 @@ declare type Props = {
   color?: string,
```

Comment:
Would be nice to avoid show telefonica icons when you set O2 brand in mistica-icons story